src/main/java/com/example/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,27 @@
 package com.example.bookstore.dto;
 
+import com.example.bookstore.validation.image.Image;
+import com.example.bookstore.validation.isbn.Isbn;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
for String validation use @NotBlank
read about the difference of NotNull and NotBlank annotations
fix other fields
```suggestion
    @NotBlank
```